variants/backend-base/bin/setup

```diff
@@ -6,7 +6,7 @@ def setup!
     run  "gem install bundler --no-document --conservative"
     run  "bundle install"
     run  "yarn install" if File.exist?("yarn.lock")
-    run  "bundle exec overcommit --install"
+    run  'bundle exec overcommit --install unless ENV["SKIP_OVERCOMMIT"]'
```

Comment:
issue: this is not actually running 😅 

```suggestion
    run  "bundle exec overcommit --install" unless ENV["SKIP_OVERCOMMIT"]
```

suggestion: it would be good to also skip this is `CI` is set

Reply:
i think we talked about bin/setup being for dev setup and not for ci, but i may have imagined the conclusion there.

Reply:
Yes we've talked about it (and might have even officially moved off using it but can't remember either) but ultimately there's nothing preventing it from happening and having overcommit run is at best annoying/slightly slower and at worse causes issues.

I'm suggesting from the POV of "its cheap, I've seen it be a pain in the past, it might save someone some pain in the future" 🙂